cpe/openscap-cpe-oval.xml

```diff
@@ -442,6 +442,20 @@
                         <criterion comment="openSUSE 42.1 is installed" test_ref="oval:org.open-scap.cpe.opensuse:tst:421"/>
                   </criteria>
             </definition>
+            <definition class="inventory" id="oval:org.open-scap.cpe.wrlinux:def:8" version="1">
+                  <metadata>
+                        <title>WR Linux 8</title>
+                        <affected family="unix">
+                            <platform>WR Linux 8</platform>
+                        </affected>
+                        <reference ref_id="cpe:/o:windriver:wrlinux:8" source="CPE"/>
+                        <description>The operating system installed on the system is WR Linux 8</description>
+                  </metadata>
+                  <criteria>
+                        <criterion comment="Installed operating system is part of the unix family" test_ref="oval:org.open-scap.cpe.wrlinux:tst:1"/>
+                        <!-- <criterion comment="WR Linux 8 is installed" test_ref="oval:org.open-scap.cpe.wrlinux:tst:8"/> -->
```

Comment:
Why is this commented out?

As it is, all operating systems of unix family will behave as if they were WR Linux in openscap CPE.